src/genezio.ts

```diff
@@ -229,12 +228,18 @@ program
     )
     .option("--language <language>", "Language of the SDK.", "ts")
     .option(
-        "-s, --source <source>",
+        "--source <source>",
+        "Where the SDK should be generated from. Possible values: local, remote.",
+        "./",
```

Comment:
```suggestion
    .addOption(
        new Option("--source <source>", "Where the SDK should be generated from.")
            .choices(["local", "remote"])
            .default("local"),
    )
```

This way, you can specifiy the possible choices and they will be handled by commander. Also you should set `local` or `remote` (not sure which, you decide) by default, not `"./"`.